.github/workflows/rust-release.yml

```diff
@@ -0,0 +1,27 @@
+on:
+  release:
+    types: [created]
+
+jobs:
+  release:
+    name: release ${{ matrix.target }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - target: x86_64-pc-windows-gnu
+            archive: zip
+          - target: x86_64-unknown-linux-musl
+            archive: tar.gz
+          - target: x86_64-apple-darwin
+            archive: zip
```

Comment:
can we add apple silicon target?

Reply:
it looks unsupported with this action
https://github.com/rust-build/rust-build.action/issues/4#issuecomment-1044914831